config/schemas/form_response.json

```diff
@@ -314,6 +317,14 @@
     "whole_number": {
       "type": "string",
       "pattern": "^[0-9]+$"
+    },
+    "how_much_charge": {
+      "type": ["string", "null"],
```

Comment:
Is `null` an option for this field? Is that if one skips this question page?

Reply:
Yes. Although I think I should have added it to the enum, rather than here.

Reply:
Apparently it's not needed, so I'm going to remove the "null" and fix up.